roomedit/include/bsp.h

```diff
@@ -56,6 +58,7 @@
 #define WF_BELOW_TOPDOWN  0x00000080      // Draw lower texture top-down
 #define WF_NORMAL_TOPDOWN 0x00000100      // Draw normal texture top-down
 #define WF_NO_VTILE       0x00000200      // Don't tile texture vertically (must be transparent)
+#define WF_CLAMP          0x00000400      // Clamp texture
```

Comment:
Heads up that this value is not safe to use because 0x400 is already used for WF_HAS_ANIMATED in the client (whatever that is), in clientd3d/bsp.h.  Please copy over that client value so that this can be avoided in the future.